src/console/ServiceInstallCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace blink\console;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ServiceInstallCommand
+ *
+ * @package blink\console
+ */
+class ServiceInstallCommand extends BaseService
+{
+    public $name = 'service:install';
+    public $description = 'Install {serviceName} as a system service';
+    public $bin = 'blink';
+
+    protected function configure()
+    {
+        $this->addOption('php', null, InputOption::VALUE_REQUIRED, 'Specify a custom php executable');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->ensureRootPrivilege();
+        $this->ensurePkgConfigInstalled();
+
+        $unitFile = $this->getSystemUnitDir() . '/' . $this->serviceName . '.service';
+
+        file_put_contents($unitFile, $a = $this->getServiceConfig($input->getOption('php')));
+
+        $envFile = '/etc/default/' . $this->serviceName;
+        if (!file_exists($envFile)) {
+            file_put_contents($envFile, $this->getEnvConfig());
+        }
+
+        system('systemctl enable ' . $this->serviceName);
```

Comment:
If the $serviceName's value contains a space or other       character need to be escaped, this line will work as expected?

Reply:
Blank characters will not work, we may consider converting these characters to `-`.